app/components/Logo.tsx

```diff
@@ -2,17 +2,17 @@ import Image from "next/image";
 
 const Logo = () => {
   return (
-    <div className="flex flex-col items-center justify-center min-w-full mb-20">
-      <div className="flex flex-col mt-[10vw]">
+    <div className="flex flex-col items-center justify-center min-w-full mb-12">
+      <div className="flex flex-col mt-[14vw] 2xl:mt-[16vw]">
```

Comment:
The intention is to keep the Sun/O in DAOcember just above the ridge. It mostly works but I feel like this could be more exact (some `calc`) or smoothed.